src/DynamicData/Binding/SortAndBind.cs

```diff
@@ -33,12 +34,17 @@ public SortAndBind(IObservable<IChangeSet<TObject, TKey>> source,
         // static one time comparer
         var applicator = new SortApplicator(_cache, target, comparer, options);
 
-        _sorted = source.Do(changes =>
+        if (options.Scheduler is not null)
+            source = source.ObserveOn(options.Scheduler);
```

Comment:
This is insufficient. Refer to #932, the logical query this needs to serve as a replacement for is...

```cs
.Sort(...)
.ObserveOn(...)
.Bind(...)
```

I.E. the scheduling needs to be applied to both input streams, `source` and `comparer`. Otherwise, notifications coming in on `comparer` can trigger the `ObservableCollection` be manipulated off of the main thread.

Another `.ObserveOn()` call applied to `comparer` should be sufficient here, since there is no change stream "between" the sorting and binding steps anymore.

Reply:
Fair call, I'll fix that.  

Also I was wondering whether to add a scheduler overloads to SortAndBind, or instead to instruct the use of

```cs
DynamicDataOptions.SortAndBind = DynamicDataOptions.SortAndBind with
{
    Scheduler = myMainThreadScheduler
};
```
which is neat but not obvious.

Or should I do both?  Thoughts?

Reply:
I've applied the scheduler to the comparer changed observable, and moved the UI thread scheduler option to `DynamicDataOptions.MainThreadScheduler`.

So the one remaining question is whether it's worth adding 12 new overloads to pass the comparer to SortAndBind or to rely on the global main thread scheduler?

Reply:
I'd say both. There's value in being able to specify on a per-call basis, and in being able to set it globally and not have to worry about it.

Not sure about whether a global-level scheduler should be universal, or per-operator. Again, I think there's value in both, but in this case, I think it's a little cumbersome to HAVE both. If it's not per-operator, I would suggest naming it `BindScheduler` because that clarifies how it's actually being USED if you set it. It would only be pulled for binding operations.

Reply:
Renamed to BindingScheduler + added scheduler overloads for new generation of binding operators.

Reply:
> Also I was wondering whether to add scheduler overloads to SortAndBind, or instead to instruct the use of

I suggest both: Giving the ability to provide all the options for their special cases AND providing a helper method that facilitates the most common use case(s).